src/Polly.Strategies/Outcome.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Polly;
+
```

Comment:
Would it make sense to put the Outcome, Events, Predicates type in the .Abstractions package? Specifically, if I were to implement new strategies, I should be able to do so by importing only the .Abstractions package and not the standard Strategies package.

Reply:
Implementation of new strategy also assumes integration with `IResilienceStrategyBuilder` which is in the Core. 

However, the `IResilienceStrategyBuilder` also pulls in more packages (`Logging.Abstractions`) that's why I thought we can keep it in Core.